index.js

```diff
@@ -1,17 +1,14 @@
-/*eslint no-var:0, prefer-arrow-callback: 0, object-shorthand: 0 */
-'use strict';
-
-
-var Punycode = require('punycode/');
+import Punycode from 'punycode';
+import rules from './data/rules.js';
 
 
 var internals = {};
```

Comment:
Podríamos también aprovechar este esfuerzo para deshacernos de las declaraciones que usan `var`?

Dime qué opinas y si quieres yo mando ese cambio!

Reply:
Totally agree, but I'd rather wait until this PR is meged to avoid making it too huge and harder to review. Can you create a new issue for this?